app/templates/components/preprint-form-authors.hbs

```diff
@@ -114,12 +121,12 @@
                                 {{#if contrib.unregisteredContributor}}
                                     {{contrib.unregisteredContributor}}
                                 {{else}}
-                                    <a href={{contrib.users.profileUrl}}> {{contrib.users.fullName}} </a>
+                                    <a href={{contrib.users.links.html}}> {{contrib.users.fullName}} </a>
```

Comment:
Hmm, that's very strange- it's a helper on the user model, and should be available as long as the relationship can be loaded. (in which case both ways of getting the data would be there?)

Not urgent, but certainly something we'll keep an eye out for.